include/segment_symbols.h

```diff
@@ -655,8 +655,8 @@ DECLARE_ROM_SEGMENT(spot20_room_0)
 
 DECLARE_ROM_SEGMENT(ganon_tou_room_0)
 
-// Room symbols for compiling test scenes, these are not wrapped in an `#if OOT_DEBUG`
-// so that debug ROMs (including gc-eu-mq-dbg) can be built with OOT_DEBUG=0.
+// Room symbols for compiling test scenes, these are not wrapped in an `#if DEBUG_FEATURES`
+// so that debug ROMs (including gc-eu-mq-dbg) can be built with DEBUG_FEATURES=0.
 
 DECLARE_ROM_SEGMENT(test01_room_0)
```

Comment:
```suggestion
// Room symbols for compiling test scenes, these are not wrapped in an `#if DEBUG_ASSETS`
// so that debug ROMs (including gc-eu-mq-dbg) can be built with DEBUG_ASSETS=0 (e.g. when DEBUG_FEATURES=0).

DECLARE_ROM_SEGMENT(test01_room_0)
```

this comment refers to symmetry with the the #if in the spec around debug maps, which is now using DEBUG_ASSETS

Reply:
I fixed wording of a similar comment above this one too, unrelated to these changes